Gemfile

```diff
@@ -1,9 +1,20 @@
 # foreman plugins import this file therefore __FILE__ cannot be used
 FOREMAN_GEMFILE = __FILE__ unless defined? FOREMAN_GEMFILE
 
+require_relative 'config/boot_settings'
+
 source 'https://rubygems.org'
 
-gem 'rails', '4.2.8'
+case SETTINGS[:rails]
+when '4.2'
+  gem 'rails', '4.2.8'
+when '5.0'
+  gem 'rails', '5.0.3'
```

Comment:
I'm not sure I understand your question. Supporting both is unlikely to be useful, but they can use the default Rails version or override it via `config/settings.yaml.dist` if desired.